addons/payment_stripe/controllers/main.py

```diff
@@ -18,15 +18,15 @@ def stripe_s2s_create_json(self, **kwargs):
         acquirer = request.env['payment.acquirer'].browse(acquirer_id)
         if not kwargs.get('partner_id'):
             kwargs = dict(kwargs, partner_id=request.env.user.partner_id.id)
-        return acquirer.s2s_process(kwargs).id
+        return acquirer.with_context(stripe_manual_payment=True).s2s_process(kwargs).id
```

Comment:
not needed in the original stripe module (the 'non-sca' one), as the routes are either completely overwritten in the sca modules or disabled entirely.